admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.js

```diff
@@ -55,6 +55,8 @@ export default class ProductPartialUpdater {
    * ex: new ProductPartialUpdater($productForm, $productFormSubmitButton).watch();
    */
   watch() {
+    // Avoid submitting form when pressing Enter
+    this.$productForm.keypress((e) => e.which !== 13);
```

Comment:
Mmm, I see

But then the problem is here because the combination components is inside the product form Which we should try and avoid Since it contains some undesired inputs that have nothing to do in the product form (they are independent)

In any other tab, since they belong to the product form pressing enter should submit the form as any other form would do no? 🤔